storage/storage_tests/storage_tests.cpp

```diff
@@ -467,43 +467,6 @@ UNIT_TEST(StorageTest_DeleteTwoVersionsOfTheSameCountry)
   TEST_EQUAL(TStatus::ENotDownloaded, storage.CountryStatusEx(countryId), ());
 }
 
-UNIT_TEST(StorageTest_DownloadCountryAndDeleteRoutingOnly)
```

Comment:
What about backward compatibility? Or in the new version of app it won't be possible to download maps without transition to small mwms?